index.html

```diff
@@ -102,130 +108,151 @@ <h3 class="work-project-header">Multi-Post Stories</h3>
               ever since the 1500s, when an unknown printer took a standard
               dummy text.
             </p>
+
+            <ul class="lang-list-ul">
+              <li class="lang-list">css</li>
```

Comment:
_This inline comment is for lines 87-89.I cant choose the title of my recent work for inline comment. So please ignore the lines that ı choose_

- [ ] Please check `my recent works title and horizontal line. ` how is it in the original design. Kindly fix it.

![image](https://user-images.githubusercontent.com/38635158/144271858-79976a52-c8d9-4cbb-a46b-db305af69937.png)
ORIGINAL
![image](https://user-images.githubusercontent.com/38635158/144271917-3e6ba8fd-d5ad-415a-8273-21ab16316e2e.png)